M2/Macaulay2/m2/galois.m2

```diff
@@ -225,6 +225,7 @@ GF(Ring) := GaloisField => opts -> (S) -> (
 	  );
      F.use = F -> var <- F_0;
      F.use F;
+     F.cache = new CacheTable;
```

Comment:
Is this just for future use?  Or are there some methods defined for ancestors that might look for it?

Reply:
It's for my personal uses :)